internal/scraper/scraper.go

```diff
@@ -39,7 +38,7 @@ func NewScraper(client HTTPClient) *Scraper {
 // Scrape extracts the recipe from the given URL. An error will be
 // returned when the URL cannot be parsed.
 func (s *Scraper) Scrape(url string, files services.FilesService) (models.RecipeSchema, error) {
-	host := getHost(url)
+	host := services.GetHost(url)
```

Comment:
This function should either 
1. Stay in the scraper package because it's built for fetching websites that require special headers to work, or
2. Have its own `interface` in the services package because every service has its [own interface](https://github.com/reaper47/recipya/blob/main/internal/services/service.go).

I would go with 2 as you have a nice catch down there in [files.go](https://github.com/reaper47/recipya/pull/353/files#diff-48c4458f9559917b24b52d38bde72473ce2a934b8791a982b3cbeee0beb3e495). I will do it if for any reason you are uncomfortable with Go interfaces.